src/CriteriaCollection.php

```diff
@@ -2,23 +2,26 @@
 
 namespace EthicalJobs\Storage;
 
-/**
- * Collection of criteria
- *
- * @author Andrew McLagan <andrew@ethicaljobs.com.au>
- */
 class CriteriaCollection extends Collection
 {
     /**
      * Push a criteria class
      *
-     * @param string $criteria
+     * Unclear what the purpose of this is, why it doesn't push but instead puts or why it was made a dependency.
+     *
+     * @TODO Remove dependency on this and potentially entire package
+     *
+     * @param  mixed  $values [optional]
      * @return $this
      */
-    public function push($criteria)
+    public function push(...$values)
```

Comment:
ive never seen ... syntax in php, does it bundle all the args into a single value?

Reply:
it's the "splat" operator used for argument unpacking https://wiki.php.net/rfc/argument_unpacking